dangerfile.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
```

Comment:
@tivac Don't feel bad; I'm not a super big fan of it myself. 😉  I'm just valuing project consistency over [my own opinions](https://github.com/isiahmeadows/eslint-config-isiahmeadows).